models/experimental/llama2_70b/tests/unit_tests/test_nlp_create_head.py

```diff
@@ -87,12 +87,12 @@ def run_test_create_head1(
         q_heads_tt,  # [seqlen, n_local_heads, bsz, head_dim]
         k_heads_tt,  # [seqlen, n_local_kv_heads, bsz, head_dim]
         v_heads_tt,  # [seqlen, n_local_kv_heads, bsz, head_dim]
-    ) = ttl.tensor.nlp_create_qkv_heads(
+    ) = ttnn.experimental.transformer.nlp_create_qkv_heads(
```

Comment:
this can be ttnn.experimental.nlp_create_qkv_heads, right?

Reply:
Yes, done